lib/compat.php

```diff
@@ -147,3 +147,175 @@ function gutenberg_wpautop( $content ) {
 }
 remove_filter( 'the_content', 'wpautop' );
 add_filter( 'the_content', 'gutenberg_wpautop', 8 );
+
+
+/**
+ * Check if we need to load the block warning in the Classic Editor.
+ *
+ * @since 3.4.0
+ */
+function gutenberg_check_if_classic_needs_warning_about_blocks() {
+	global $pagenow;
+
+	if ( ! in_array( $pagenow, array( 'post.php', 'post-new.php' ), true ) || ! isset( $_REQUEST['classic-editor'] ) ) {
+		return;
+	}
+
+	if ( isset( $_REQUEST['hide-block-warning'] ) ) {
+		return;
+	}
+
+	$post = get_post();
+	if ( ! $post ) {
+		return;
+	}
+
+	if ( ! gutenberg_post_has_blocks( $post ) ) {
+		return;
+	}
+
+	wp_enqueue_script( 'wp-sanitize' );
```

Comment:
At first glance, I wasn't clear what this was here for (could do for code comment). I eventually found the use in inline script below. Appears then we'd also need to enqueue the dependency making `wp.a11y` available?